tests/code-editor.spec.ts

```diff
@@ -0,0 +1,248 @@
+import { test, expect } from '@playwright/test';
+
+test('Renders JSON code editor correctly', async ({ page }) => {
+  await page.goto('/code-editor-test.html');
+  const validJSONEditor = page.getByTestId('code-editor-json-valid')
+  await expect(validJSONEditor).toHaveCount(1)
```

Comment:
So actually the visibility check doesn't work here because the div has width 0 x height 0, but i just instead removed these checks because they were mainly to make sure that the container existed but if the attribute check passes then it implicitly states that it does.

Reply:
👍 Makes sense